docroot/modules/custom/json_data_field/src/Plugin/Validation/Constraint/ValidJsonSchemaConstraintValidator.php

```diff
@@ -35,8 +35,9 @@ public function validate($value, Constraint $constraint) {
     $response_data = json_decode($data);
 
     // Get the json_data_field JsonSchema file.
+    /* @phpstan-ignore-next-line */
     $field_storage_settings = $value->getFieldDefinition()->getSettings();
-
+    /* @phpstan-ignore-next-line */
```

Comment:
As $value is not defined in the method, that's the reason it is throwing phpstan error. If we add variable comment to define $value, it should work fine but i'm not sure $value is of which type.

Reply:
Resolved.